src/language/server.ts

```diff
@@ -61,11 +66,16 @@ connection.onInitialize((params: InitializeParams) => {
           },
         },
       },
-      // Tell the client that the server supports code completion
+      // Tell the client that the server supports code completion.
       completionProvider: {
         resolveProvider: true,
         triggerCharacters: ['.'],
       },
+      // Tell the client that the server supports help signatures.
+      signatureHelpProvider: {
+        resolveProvider: true,
+        triggerCharacters: [',', '('],
+      },
```

Comment:
Just checking, is this a leftover from before the .d.ts change or is it still having an effect?

Reply:
Actually, we still need it, otherwise the `connection.onSignatureHelp` won't be called and help signatures won't be returned from the ts service.